### ISSUE TITLE
Request: Remove dependence on external resources

### ISSUE THREAD

**Issue by aujawindar** [yursan9/manis-hugo-theme#25]
The file layouts/partials/meta.html contains links to external fonts (from googleapis.com) and css/js (from Cloudflare). It would be nicer to have relevant features and fonts locally instead of relying on external services. I believe this is also more in line with Hugo sites and philosophy in general.

**Comment by yursan9** [comment]
But I need the font and css/js? Or you think I should download the font, css, and js and include it with the theme?

**Comment by aujawindar** [comment]
Absolutely, that would be a very good move. This is also more in line with current development within EU, if that matters. Linking to Google fonts is no longer recommended.

**Comment by yursan9** [comment]
Well I'm happy if you can provide the PR, I am not EU citizen, and this theme can be use and modify by anyone if they need to comply with local regulation.
